direct/src/interval/IntervalManager.py

```diff
@@ -133,7 +133,7 @@ def __doPythonCallbacks(self):
     def __storeInterval(self, interval, index):
         while index >= len(self.ivals):
             self.ivals.append(None)
-        assert self.ivals[index] == None or self.ivals[index] == interval
+        assert self.ivals[index] is None or self.ivals[index] == interval
```

Comment:
Note that any class that is wrapped by interrogate *must* use `==` since two classes with the same underlying C++ pointer may still not match via `is`.

I would suggest leaving this case alone if we're not sure.